windu.el

```diff
@@ -58,7 +58,7 @@
             (define-key map (kbd "{") 'windu-order-height-bottom)
             (define-key map (kbd "}") 'windu-order-height-top)
             ;; Splits / Group Orders
-            (define-key map (kbd "3") 'windu-split-window-best-effort)
+            (define-key map (kbd "b") 'windu-split-window-best-effort)
```

Comment:
Do you use this, or any of the other split-window functions?  I personally only use `windu-split-window-best-effort`, and I don't use this key combo for it.  (I rebound `C-x 3` so I just always use it when splitting left/right).